File: python/questao9.py
def main():
    casa = float(input("Qual o valor da casa? "))
    salario = float(input("Qual o valor do salário? "))
    anos = float(input("Em quantos anos deseja pagar? "))

    prestacao = casa / (anos * 12)

    prestacao_maxima = salario * 0.3
    if prestacao > prestacao_maxima:
        print("O empréstimo não foi aprovado.")
    else:
        print("Aprovado!")

if __name__ == "__main__":
    main()

File: c/questao9.c
#include <stdio.h>

int main() {
    float casa, salario, anos, prestacao;

    printf("Qual o valor da casa? ");
    scanf("%f", &casa);

    printf("Qual o valor do salário? ");
    scanf("%f", &salario);

    printf("Em quando anos deseja pagar? ");
    scanf("%f", &anos);

    prestacao = casa / (anos * 12);

    float prestacaoMaxima = salario * 0.3;
    if(prestacao > prestacaoMaxima) {
        printf("O empréstimo não foi aprovado.\n");
    } else {
        printf("Aprovado!");
    }

    return 0;
}